.github/workflows/run_tests.yml

```diff
@@ -11,14 +11,24 @@ on:
     - master
 
 jobs:
-  build:
+  create_project:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v1
     - name: Generate project
       run: npx create-react-app tmp --template file:.
+    - name: Use plop
+      run: |
+        cd tmp
+        yarn plop component test ./src/shared/components
+        yarn plop module test
+        yarn plop hook test ./src/shared/components
```

Comment:
Tests for new created component will fail without `--u` flag because the snapshots were not created yet.

I think we need to run tests on clean project first -> create files with plop -> run tests again with `--u` flag to ignore snapshots this time